ephypype/tests/test_import_ctf.py

```diff
@@ -0,0 +1,15 @@
+"""Test import_ctf."""
+import mne
+import os.path as op
+
+from ephypype.import_ctf import convert_ds_to_raw_fif  # noqa
+
+data_path = mne.datasets.testing.data_path()
+ds_fname = op.join(data_path, 'CTF', 'testdata_ctf.ds')
+
+
+def test_convert_ds_to_raw_fif():
+    """Test data conversion."""
+
+    raw_fif_file = convert_ds_to_raw_fif(ds_fname)
```

Comment:
The working directory is where you run the test from. Why do you need it?

Reply:
since some ephypype functions save files using the abspath. 
Following your suggestion I created a tmp dir 
https://github.com/neuropycon/ephypype/blob/d9f7ec2ed24a0379af9327c7a2400582e4bd4323/ephypype/tests/test_fif2ts.py#L20